packages/opencensus-core/src/common/validations.ts

```diff
@@ -37,8 +37,9 @@ export function validateNotNull<T>(reference: T, errorMessage: string): T {
  */
 export function validateArrayElementsNotNull<T>(
     array: T[], errorMessage: string) {
-  if (array.every(
-          element => element !== null || typeof element !== 'undefined')) {
+  const areAllNotNullOrUndefined = array.every(
+      element => element !== null && typeof element !== 'undefined');
+  if (!areAllNotNullOrUndefined) {
```

Comment:
would you mind to rename it to `areAllDefined`?
in this case it will be more simple to read

Reply:
SGTM, Done.